package.json

```diff
@@ -11,9 +11,9 @@
   },
   "scripts": {
     "start": "node index.js",
-    "browserify": "browserify -r ./lib/kubebox.js:kubebox -r ./lib/http-then.js:http-then -r blessed -i pty.js -o docs/kubebox.js",
-    "browserify-debug": "browserify --debug -r ./lib/kubebox.js:kubebox -r ./lib/http-then.js:http-then -r blessed -i pty.js | exorcist docs/kubebox.js.map > docs/kubebox.js",
-    "bundle": "browserify index.js -o bundle.js -i pty.js --bare",
+    "browserify": "browserify -r ./lib/kubebox.js:kubebox -r ./lib/http-then.js:http-then -r blessed -i pty.js -i node-pty.js -i jsdom.js -i xterm.js -o docs/kubebox.js",
+    "browserify-debug": "browserify --debug -r ./lib/kubebox.js:kubebox -r ./lib/http-then.js:http-then -r blessed -i pty.js -i node-pty.js -i jsdom.js -i xterm.js| exorcist docs/kubebox.js.map > docs/kubebox.js",
+    "bundle": "browserify index.js -o bundle.js -i pty.js -i node-pty.js -i jsdom.js -i xterm.js --bare",
```

Comment:
It seems there is a 300KB increase in bundle sizes. Wonder what contributes to that increase...